ui/cypress/e2e/communityTemplates.test.ts

```diff
@@ -226,15 +226,9 @@ describe('Community Templates', () => {
       cy.go('back')
     })
 
-    it('takes you to github when you click on the Community Templates link', () => {
-      cy.getByTestID('template-source-link').within(() => {
-        cy.contains('Community Templates').should(
-          'have.attr',
-          'href',
-          'https://github.com/influxdata/community-templates/blob/master/docker/docker.yml'
-        )
-      })
-      //TODO: add the link from CLI
+    it('takes you to github readme when you click on the Community Templates button', () => {
```

Comment:
we'll want to keep an eye on this. these tests actually make GET requests to github, which might be against TOS if we do it in an automated test.

Reply:
good to know, i did not know about Githubs TOS